src/components/TopbarViewer/TopbarViewer.style.tsx

```diff
@@ -43,7 +43,7 @@ export const StyledSearchbar = styled(Searchbar)<FocusProps>`
   margin-left: auto;
```

Comment:
I cannot comment here, and I know that search is being reworked but I think we could change line 42 to: 
```tsx
width: ${({ hasFocus }) => (hasFocus ? '100%' : '39px')};
```
Right now there is unnecessary space between the input and hamburger button.
Design:
![image](https://user-images.githubusercontent.com/51168865/136151812-737a4632-6736-42f9-9173-7db7e406b530.png)
Ours: 
![image](https://user-images.githubusercontent.com/51168865/136151869-76f5fa2e-388d-48d9-8ae0-f0fd900dd5cc.png)

Reply:
Done